website/docs/r/kubernetes_cluster.html.markdown

```diff
@@ -136,6 +136,10 @@ resource "azurerm_subnet" "virtual" {
 
 * `windows_profile` - (Optional) A `windows_profile` block as defined below.
 
+* `private_link_enabled` (Optional) Enables private link for the AKS cluster, providing a private IP for the k8s api. Defaults to false.
+
+-> **NOTE:**  At this time Private Link is in Public Preview. For an example of how to enable a Preview feature, please visit  [Private Azure Kubernetes Service cluster] (https://docs.microsoft.com/en-gb/azure/aks/private-clusters)
```

Comment:
minor formatting so this renders correctly here:

```suggestion
-> **NOTE:**  At this time Private Link is in Public Preview. For an example of how to enable a Preview feature, please visit [Private Azure Kubernetes Service cluster](https://docs.microsoft.com/en-gb/azure/aks/private-clusters)
```

Reply:
Resolved in new commit